srcpkgs/atom-bin/REMOVE

```diff
@@ -0,0 +1,9 @@
+if [ "$UPDATE" = "no" ]; then
+    case "$ACTION" in
+        post)
+            if [ -d usr/share/atom ]; then
+                rm -rf usr/share/atom
+            fi
+        ;;
+    esac
+fi
```

Comment:
Why this?

Reply:
Because the directory does get removed otherwise.

Reply:
This is a bug: https://github.com/void-linux/xbps/issues/23
and don't have the REMOVE file just to fix this. xbps will be fixed to handle it.

Reply:
Nice to know. Thanks.